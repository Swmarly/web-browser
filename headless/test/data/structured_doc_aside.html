<!DOCTYPE html>
<html lang="en">
<body>
<aside>
Content in html aside.
</aside>
<div role=complementary>
Content in ARIA role complementary.
</div>
</body>
</html>
