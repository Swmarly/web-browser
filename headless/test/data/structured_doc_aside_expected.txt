{
   "lang": "en",
   "type": "Document",
   "~children": [ {
      "type": "Aside",
      "~children": [ {
         "type": "NonStruct"
      } ]
   }, {
      "type": "Aside",
      "~children": [ {
         "type": "NonStruct"
      } ]
   } ]
}
